### PR TITLE
submitting python-louvain

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

**Comment by ericmjl** [comment]
cc @MridulS

**Comment by ericmjl** [comment]
This PR closes https://github.com/ericmjl/Network-Analysis-Made-Simple/issues/46

**Comment by ericmjl** [comment]
ping @ocefpaf @jakirkham, I believe this PR is done.

**Comment by ericmjl** [comment]
Please let me know if I missed something!

File: recipes/python-louvain/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  # url: https://github.com/simplejson/{{ name }}/archive/{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/python-louvain/meta.yaml]: Can you remove these comments?

Comment (ericmjl) [reply recipes/python-louvain/meta.yaml]: No problem!

File: recipes/python-louvain/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (ocefpaf) [inline recipes/python-louvain/meta.yaml]: Can this package be noarch? Looks like it is possible but I did not see the code to check if there are compiled parts or some automatic py2/3 conversion in `setup.py`.

Comment (ericmjl) [reply recipes/python-louvain/meta.yaml]: Looks like so. There's no Cython, no 2/3 conversions, so I'm going to make this happen then.

**Comment by ericmjl** [comment]
@conda-forge-admin, please add noarch: python

**Pull request merged**